render-only-sample/rosumd/RosUmdResource.cpp

```diff
@@ -331,7 +338,16 @@ RosUmdResource::CalculateMemoryLayout(
                 assert(m_mipLevels == 1);
 
                 assert((m_hwFormat == RosHwFormat::X8888) || (m_hwFormat == RosHwFormat::D24S8));
-                UINT sizeTileBytes = 64 * 64 * 4;
+                
+                // Values are hardocded - we are using RosHwFormat::X8888 format
+                m_hwWidthTilePixels = 32;
+                m_hwHeightTilePixels = 32;
+                m_hwWidthTiles = (m_hwWidthPixels + m_hwWidthTilePixels - 1) / m_hwWidthTilePixels;
```

Comment:
Hi, Marek:

     Instead of 32, I think it is better we use VC4_4KB_TILE_WIDTH and VC4_4KB_TILE_HEIGHT, so that you can later on easily find what need to be change when more formats are supported.

                                     Thanks, Indy, 2/9/2016